terror_error.go

```diff
@@ -31,6 +31,66 @@ type ErrCodeText string
 type ErrorID string
 type RFCErrorCode string
 
+var class2rfcCode = map[int]string{
+	1:  "autoid",
+	2:  "ddl",
+	3:  "domain",
+	4:  "evaluator",
+	5:  "executor",
+	6:  "expression",
+	7:  "admin",
+	8:  "kv",
+	9:  "meta",
+	10: "planner",
+	11: "parser",
+	12: "perfschema",
+	13: "privilege",
+	14: "schema",
+	15: "server",
+	16: "struct",
+	17: "variable",
+	18: "xeval",
+	19: "table",
+	20: "types",
+	21: "global",
+	22: "mocktikv",
+	23: "json",
+	24: "tikv",
+	25: "session",
+	26: "plugin",
+	27: "util",
+}
+
+var rfcCodeToClass = map[string]int{
```

Comment:
Prefer to initialize this map in the `init` function and the base on `class2rfcCode`.